<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Yeastar-PC" name="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Outbound" tests="17" failures="3" timestamp="26 Oct 2018 03:30:25 GMT" time="519.529" errors="0">
  <testcase name="E_timecondition" time="96.516" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Outbound">
    <failure type="java.lang.AssertionError" message="预期1000为Talking状态 【预期：3 实际：1】">
      <![CDATA[java.lang.AssertionError: 预期1000为Talking状态 【预期：3 实际：1】
	at com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1092)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.failNotEquals(YsAssert.java:583)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEqualsImpl(YsAssert.java:146)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:120)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:429)
	at com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Outbound.E_timecondition(Outbound.java:332)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </failure>
  </testcase> <!-- E_timecondition -->
  <testcase name="D2_rrmemory_hunt" time="21.720" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Outbound">
    <failure type="java.lang.AssertionError" message="预期1000为Talking状态 【预期：3 实际：1】">
      <![CDATA[java.lang.AssertionError: 预期1000为Talking状态 【预期：3 实际：1】
	at com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1092)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.failNotEquals(YsAssert.java:583)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEqualsImpl(YsAssert.java:146)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:120)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:429)
	at com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Outbound.D2_rrmemory_hunt(Outbound.java:275)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </failure>
  </testcase> <!-- D2_rrmemory_hunt -->
  <testcase name="A0_init" time="14.382" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Outbound"/>
  <testcase name="A_out1_sip" time="37.819" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Outbound"/>
  <testcase name="B2_Prepend" time="41.826" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Outbound"/>
  <testcase name="C2_password" time="36.459" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Outbound">
    <failure type="java.lang.AssertionError" message="CDR_Status检测 【预期：Answered 实际：No Answer】">
      <![CDATA[java.lang.AssertionError: CDR_Status检测 【预期：Answered 实际：No Answer】
	at com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1092)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.failNotEquals(YsAssert.java:583)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEqualsImpl(YsAssert.java:146)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:120)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:203)
	at com.yeastar.swebtest.driver.YSMethod.YS_Extension.checkCDR(YS_Extension.java:290)
	at com.yeastar.swebtest.driver.YSMethod.YS_Extension.checkCDR(YS_Extension.java:278)
	at com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Outbound.C2_password(Outbound.java:240)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </failure>
  </testcase> <!-- C2_password -->
  <testcase name="A_out6_bri" time="0.054" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Outbound"/>
  <testcase name="B1_Prepend" time="32.839" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Outbound"/>
  <testcase name="C1_password" time="48.906" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Outbound"/>
  <testcase name="A_out4_spx" time="0.036" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Outbound"/>
  <testcase name="A_out7_E1" time="0.017" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Outbound"/>
  <testcase name="A_out5_fxo" time="0.014" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Outbound"/>
  <testcase name="A_out2_iax" time="0.053" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Outbound"/>
  <testcase name="A_out3_sps" time="53.606" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Outbound"/>
  <testcase name="D1_rrmemory_hunt" time="55.209" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Outbound"/>
  <testcase name="A_out8_gsm" time="0.020" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Outbound"/>
  <testcase name="F_delete" time="80.053" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Outbound"/>
</testsuite> <!-- com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Outbound -->
